android/BOINC/app/src/main/java/edu/berkeley/boinc/ProjectDetailsFragment.kt

```diff
@@ -364,9 +364,7 @@ class ProjectDetailsFragment : Fragment() {
                 val slideshowBinding = ProjectDetailsSlideshowImageLayoutBinding.inflate(layoutInflater)
                 var bitmap = image.image!!
                 if (scaleImages(bitmap.height, bitmap.width)) {
-                    bitmap = Bitmap.createScaledBitmap(image.image!!,
-                            image.image!!.width * 2,
-                            image.image!!.height * 2, false)
+                    bitmap = bitmap.scale(bitmap.width * 2, bitmap.height * 2, filter = false)
```

Comment:
Why do we do upscale here?

Reply:
I believe the original developer did it to make the slideshow images more visible.